src/libcore/intrinsics.rs

```diff
@@ -293,7 +293,7 @@ extern "rust-intrinsic" {
     ///     }
     /// }
     /// ```
-    #[unstable(feature = "core")]
+    #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
I felt that `intrinsics::copy_memory` was more telling than `intrinsics::copy`, but laziness also kicked in as a desire to not deal with `#[cfg(stage0)]`